newsletters/20190807-ingress.md

```diff
@@ -0,0 +1,86 @@
+Today we talk about the Kubernetes Ingress — the component in charge of routing the traffic to your apps.
+
+Here are a few interesting facts about the Ingress.
+
+## The most misunderstood components of the cluster.
```

Comment:
To make it sound more like the other items:

`## It's the most misunderstood component of the cluster`